packages/plugins/Solana/src/locales/index.ts

```diff
@@ -0,0 +1,30 @@
+// This file is auto generated. DO NOT EDIT
+// Run `npx gulp sync-languages` to regenerate.
+// Default fallback language in a family of languages are chosen by the alphabet order
+// To overwrite this, please overwrite packages/scripts/src/locale-kit-next/index.ts
+import en_US from './en-US.json'
+import es_ES from './es-ES.json'
+import fa_IR from './fa-IR.json'
+import fr_FR from './fr-FR.json'
+import it_IT from './it-IT.json'
+import ja_JP from './ja-JP.json'
+import ko_KR from './ko-KR.json'
+import qya_AA from './qya-AA.json'
+import ru_RU from './ru-RU.json'
+import zh_CN from './zh-CN.json'
+import zh_TW from './zh-TW.json'
```

Comment:
We don't have the following languages currently. Please remove them and run `npx gulp sync-languages` again:

- es-ES
- fa-IR
- fr-FR
- it-IT
- ru-RU

Reply:
Done.